src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,93 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
```

Comment:
constanta v 90% pripadech velkym takze PATH + path k cemu aspon. PATH_FILE_RESULTS ... etc.

Reply:
Dobré vědět, toto mi ještě dá zabrat, ale zkusím na to myslet